classes/filter/Filter.inc.php

```diff
@@ -418,10 +418,12 @@ function isCompatibleWithRuntimeEnvironment() {
 	 *
 	 * @param mixed an input value that is supported
 	 *  by this filter
+	 * @displayErrors boolean weather the value
```

Comment:
Syntax should be

    @param $displayErrors boolean ...description...

(The other param is also described incorrectly.)

Also, maybe better to call it `returnErrors` rather than `displayErrors`.